pkg/init/cmd/service/runc.go

```diff
@@ -50,42 +51,78 @@ func runcInit(rootPath string) int {
 			status = 1
 			continue
 		}
+
 		pidfile := filepath.Join(tmpdir, name)
 		cmd := exec.Command(runcBinary, "create", "--bundle", path, "--pid-file", pidfile, name)
 
 		stdoutFile := filepath.Join("/var/log", name+".out.log")
-		stderrFile := filepath.Join("/var/log", name+".err.log")
 
-		stdoutFd, err := os.OpenFile(stdoutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+		stdoutFd, stdoutFdErr := os.OpenFile(stdoutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
 		defer stdoutFd.Close()
-		if err == nil {
-			cmd.Stdout = stdoutFd
+
+		var stdoutCopyErr error
+
+		if stdoutFdErr == nil {
+			stdoutIn, _ := cmd.StdoutPipe()
+			stdoutWriter := io.MultiWriter(os.Stdout, stdoutFd)
+			go func() {
+				_, stdoutCopyErr = io.Copy(stdoutWriter, stdoutIn)
+			}()
```

Comment:
if a stdout log file can be opened for write/append, then io.Copy the cmd.StdoutPipe to a io.MultiWriter (of both the os.Stdout and the stdout log file)